wasm.md

```diff
@@ -212,6 +212,7 @@ To resolve these `identifiers` into concrete `indices`, some grammar production
  // ----------------------------------------------------------------
     rule #ContextLookup(IDS:Map, I:Int) => I
     rule #ContextLookup(IDS:Map, ID:Identifier) => {IDS [ ID ]}:>Int
+      requires ID in_keys(IDS)
```

Comment:
This came up a lot during testing: the function would succeed, with result `#Bottom`, get cast to `Int` and the error message was not helpful. With this change, at least execution halts when for some reason this doesn't hold (which it should if you're calling this function).